src/utils/events/formatEventToServer.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Normalizes the event object being sent to the API server
+ *
+ * @param {Event} input
+ *
+ * @returns {Object} output
+ * @returns {string} [output.allow_others_to_trigger]
```

Comment:
Just wanted to confirm but this isn't a boolean?

Reply:
Same question about `is_public`

Reply:
I think in the factory you have both of these as boolean?

Reply:
Yeah I got a little too copy-paste happy yesterday. I'll make sure to correct these to `boolean` thx :)